recipe/meta.yaml

```diff
@@ -1,18 +1,22 @@
-{% set version = "1.0.3" %}
+{% set version = "2.32.1" %}
 
 package:
     name: libuuid
     version: {{ version }}
 
 source:
     fn: libuuid-{{ version }}.tar.gz
-    url: https://sourceforge.mirrorservice.org/l/li/libuuid/libuuid-{{ version }}.tar.gz
-    sha256: 46af3275291091009ad7f1b899de3d0cea0252737550e7919d17237997db5644
+    url: https://mirrors.edge.kernel.org/pub/linux/utils/util-linux/v2.32/util-linux-{{ version }}.tar.gz
```

Comment:
The hardcoded v2.32 here won't work well for the not.

Reply:
Fixed.